configs/config.novacustom_ns5x

```diff
@@ -1,17 +1,18 @@
 CONFIG_LOCALVERSION="v1.3.0"
-CONFIG_VENDOR_CLEVO=y
+CONFIG_VENDOR_NOVACUSTOM=y
+CONFIG_MAINBOARD_VERSION="2.2A"
 CONFIG_FMDFILE="src/mainboard/clevo/tgl-u/vboot-rwa.fmd"
-CONFIG_MAINBOARD_VENDOR="Notebook"
 # CONFIG_CONSOLE_SERIAL is not set
+# CONFIG_ONBOARD_VGA_IS_PRIMARY is not set
 CONFIG_VBOOT=y
-CONFIG_BOARD_CLEVO_NS50MU=y
+CONFIG_MAINBOARD_SMBIOS_PRODUCT_NAME="NS50_70MU"
 CONFIG_CONSOLE_CBMEM_BUFFER_SIZE=0x20000
 CONFIG_EDK2_BOOT_TIMEOUT=2
-CONFIG_MAINBOARD_SMBIOS_PRODUCT_NAME="NS50_70MU"
 CONFIG_PCIEXP_HOTPLUG_BUSES=42
 CONFIG_PCIEXP_HOTPLUG_MEM=0xc200000
 CONFIG_PCIEXP_HOTPLUG_PREFETCH_MEM=0x1c000000
```

Comment:
Do we need hotplug? What for?

Reply:
I think it's based on https://github.com/Dasharo/coreboot/blob/clevo/develop/configs/config.clevo_ns50mu and we had it here as well.

I know it does not answer **why** we have it there

Reply:
@miczyg1 @macpijan
Actually based on https://github.com/Dasharo/coreboot/blob/clevo/develop-adl-p/configs/config.clevo_ns70pu.

These configs are selected by `SOC_INTEL_ENABLE_USB4_PCIE_RESOURCES` https://github.com/Dasharo/coreboot/blob/clevo/develop-adl-p/src/soc/intel/alderlake/Kconfig#L213 which is enabled for all Alderlake Clevo platforms https://github.com/Dasharo/coreboot/blob/clevo/develop-adl-p/src/mainboard/clevo/adl-p/Kconfig#L22